api/files/serializers.py

```diff
@@ -47,6 +47,14 @@ def to_internal_value(self, data):
         except StopIteration:
             self.fail('invalid_data')
 
+    def to_representation(self, value):
+
+        url = super(CheckoutField, self).to_representation(value)
+
+        ret = {'links': {'related': {'href': url, 'meta': {}}}}
```

Comment:
It seems like we're building this structure very similar to the above in a few different places, probably not just in this PR. Might be worth consolidating.
https://github.com/CenterForOpenScience/osf.io/pull/4522/files#diff-4b396d38a0947821679352d308445c5aR179
https://github.com/CenterForOpenScience/osf.io/pull/4522/files#diff-4b396d38a0947821679352d308445c5aR171